lib/tasks/populate_postcodes_table.rake

```diff
@@ -0,0 +1,26 @@
+require "json"
+
+desc ""
+task :populate_postcodes_table do
+  CSV.open("./postcodes.csv", "wb") do |csv|
+    csv << %w[id postcode results created_at updated_at]
+
+    index = 1
+    CSV.foreach("./mapit.csv") do |row|
+      postcode, locations = row
+      locations = locations.split(",")
+
+      results = {
+        "DPA" => {
+          "LNG" => locations[0].tr("()", "").to_f,
+          "LAT" => locations[1].tr("()", "").to_f,
+          "POSTCODE" => postcode,
+        },
+      }
```

Comment:
Mapit uses a different algorithm for determining Lat/Long, IIRC. 🤔 

I think it would be easiest (and most consistent / safest) to just import the postcode, and not worry about anything else. I.e. `Postcode.create(postcode: postcode, results: "{}")`. Then we can let the `ProcessPostcodeWorker` do its work.

Reply:
Yes, makes sense since the api is not used yet and we don't need all the information. 👍